pom.xml

```diff
@@ -25,7 +25,7 @@
         <maven.compiler.target>${jdkVersion}</maven.compiler.target>
 
         <!-- Versioning -->
-        <version.number>1.12.3</version.number>
+        <version.number>1.12.4</version.number>
```

Comment:
Well, since current update 1.12.3, you'll have to bump version, right?

But I can revert it to 1.12.3 if you want to.